deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-role.yaml

```diff
@@ -33,3 +33,9 @@ rules:
       - "/metrics"
     verbs:
       - get
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - '*'
```

Comment:
I wonder if this should be a role bound to the `kube-system` namespace only. This feature only needs to create leases in the same namespace, right?

Reply:
Agree! I updated already the manifests for Beats -> https://github.com/elastic/beats/pull/24958